onnx_tf/handlers/backend/non_max_suppression.py

```diff
@@ -8,7 +8,18 @@
 @onnx_op("NonMaxSuppression")
 class NonMaxSuppression(BackendHandler):
 
-  result = None
+  @classmethod
+  def get_req_vars_template(cls):
+    """ Get required variables template.
+
+    :return: Dict.
+    """
+    return {
+        'NonMaxSuppressionResult': [
```

Comment:
Just a nit, consider making a class variable for the variable name, instead of using strings in multiple places. Also suggest using common naming convention for variables, with all lower case  + underscore characters.